src/video.cpp

```diff
@@ -130,7 +130,7 @@ class swdevice_t : public platf::hwdevice_t {
     return 0;
   }
 
-  int set_frame(AVFrame *frame, AVBufferRef *hw_frames_ctx) {
+  int set_frame(AVFrame *frame, AVBufferRef *hw_frames_ctx, AVPixelFormat target_format) {
```

Comment:
changing `AVPixelFormat` here to `int` (or perhaps updating in all other cases to `AVPixelFormat`) should to fix the issue with software rendering.